aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -523,11 +531,18 @@ dotnet add GrpcGreeterClient.csproj package Grpc.Tools
 
 ### Create the Greeter client
 
-Build the client project to create the types in the `GrpcGreeter` namespace. The `GrpcGreeter` types are generated automatically by the build process.
+* Build the client project to create the types in the `GrpcGreeterClient` namespace.
 
-Update the gRPC client `Program.cs` file with the following code:
+The `GrpcGreeterClient` types are generated automatically by the build process. The tooling package [Grpc.Tools](https://www.nuget.org/packages/Grpc.Tools/) generates the following files based on the *greet.proto* file:
```

Comment:
Missing note here

Reply:
Good catch!  Thanks, I should have added it to previous .net versions in the doc as well, which is in now and corrected.